ci/scripts/bootstrap.sh

```diff
@@ -15,7 +15,10 @@ elif [[ "${BINARY_TYPE}" == "rpm" ]]; then
 elif [[ "${BINARY_TYPE}" == "exe" ]]; then
     source ci/scripts/exe/bootstrap_envvars.sh
 
-    echo "--> Install Software via Choco"
+    echo "--> Move Chocolatey cpack out of the way"   
+    mv /c/ProgramData/chocolatey/bin/cpack /c/ProgramData/chocolatey/bin/choco_cpack
```

Comment:
Is this still required on the CI?

Reply:
yes it's required. Alternative would be to specific the path to cpack before the chocolatey path, but this sounded the easiest way